pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -119,6 +119,36 @@ public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
                     + "LEFT JOIN kit_type ktype ON (request.kit_type_id = ktype.kit_type_id)  "
                     + "WHERE ex.ddp_participant_exit_id is null ";
 
+    public static final String SQL_SELECT_KIT_REQUEST =
```

Comment:
reverted to old code so that NO impact to All other existing functionality like kit search